### DISCUSSION TITLE
Asking for adbc java demo

### DISCUSSION THREAD

**Discussion by Stongtong** [duckdb/duckdb#9958] [Q&A]
I use duckdb by java in project, storing more than billion records in it, I eagerly want to know how to use adbc by java way on duckdb, I look forward to hearing from you!

**Comment by Mause** [comment]
Have you looked at the ADBC documentation? https://arrow.apache.org/adbc/main/driver/jdbc.html

You may have to use the binary blob from our python distribution though, as I'm not sure we export the ADBC symbols elsewhere

**Reply by Stongtong** [reply]
thank you for your reply , I saw it, and read duckdb adbc api in its website but there not corresponding java interface, I download entire arrow adbc source code and found adbc driver for java, unfortunately not more demo case
